README.md

```diff
@@ -10,7 +10,7 @@ A fact checker developed in Python.
 - For setting up sopex visit [sopex](https://github.com/codemaniac/sopex).
 - Now place the three files (script.py, test.txt) in the directory inside sopex directory.
```

Comment:
@sarthak5 
can you please replace the three here with two as there are two arguments inside the brackets.